src/components/Form.jsx

```diff
@@ -0,0 +1,36 @@
+import { useContext, useState } from "react";
+import plus from "../assets/plus.svg";
+import todoContext from "../todoContext";
+
+export default function Form() {
+  const { createNewTodo} = useContext(todoContext)
+  const [text, setText] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (text.trim()) {
+        createNewTodo(text.trim())
+        setText("");
+    }
+  }
```

Comment:
I'd suggest first checking the `text` value before calling the `trim` function on it. Maybe use the optimal chaining to check if the text is defined to prevent any error that might rise from this, like runtime errors. Check how I think you can improve this: 
``` suggestion
  const handleSubmit = (event) => {
    event.preventDefault();
    text?.trim() && createNewTodo(text.trim());
    setText("");
  }
```